### PR TITLE
Po/2765 update android sdk

### PR DESCRIPTION
**Pull request by poborin**
to test 

https://install.appcenter.ms/orgs/bluedot/apps/BDReactNativeIntegration-1/releases/30

### CODE REVIEW

**Review by duncan-lau — APPROVED** [review]

File: android/src/main/java/io/bluedot/AppTempoReceiver.java [hunk 1]
```diff
@@ -1,19 +1,21 @@
 package io.bluedot;
 
 import android.content.Context;
+
```
Comment (jsprincep) [inline android/src/main/java/io/bluedot/AppTempoReceiver.java:4]: 🤔 Looks like you have a different import code style which is causing a lot of unrelated changes? What IDE are you using?

Comment (poborin) [reply android/src/main/java/io/bluedot/AppTempoReceiver.java:4]: android Studio 4.1.2

Comment (jsprincep) [reply android/src/main/java/io/bluedot/AppTempoReceiver.java:4]: Ok, cool, we have a code style file here: https://gitlab.com/bluedotio/android/dev_resources_android/-/blob/master/Bluedot.xml you can dl and import, which should reduce the amount of import moving noise between MRs, amongst other benefits. Let me know if any issues.

Comment (poborin) [reply android/src/main/java/io/bluedot/AppTempoReceiver.java]: I applied the scheme and reoptimised imports. Please check again.

**Review by jsprincep — APPROVED** [review]
Nice work cleaning up the unused imports :+1:
